### DISCUSSION TITLE
post/bluetooth-remittance/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#42] [Announcements]
# post/bluetooth-remittance/

처음 만난 회사 사람과 식사.. 정산은 어떻게 하지? 아하, 내 주변 송금!

https://tech.kakaopay.com/post/bluetooth-remittance/

**Comment by pay-dan-dy** [comment]
유용하고 재밌는 기능인데 친숙한 블루투스 기술을 이용한 거였군요! 좋은 글 감사합니다~
